lib/List/styles.scss

```diff
@@ -18,6 +18,12 @@ $list-border: 1px solid $color-border-base;
   .is-active & svg path {
     fill: $color-text-base;
   }
+
+  &.is-active {
```

Comment:
This was in the item row stylesheet but had very little to do with it. I moved it into here 👍